modules/data-loading/pages/data-loading-ha.adoc

```diff
@@ -1,8 +1,12 @@
-= Data Loading High Availability (HA)
+= Data Loading High Availability (HA) BETA
 :page-aliases:
 
+NOTE: This is currently a beta feature
+
 Data Loading High Availability (HA) provides load balancing and automatic fail over in the event of a service disruption.
 
+The new data loading mode Data Loading V2 (Beta) provides support for the HA feature.
+
 == `Setup`
 First, set up a cluster with replication factor greater than 1 following xref:cluster-and-ha-management:ha-cluster.adoc[High Availability Cluster Configuration]
```

Comment:
@RyanYunboL  Are there any user privileges required to enable Kafka Connect V2? If yes, please explicitly mention it.

Reply:
No, it is a system config set by `gadmin config set`